core/src/main/java/com/decibeltx/studytracker/core/model/Statistics.java

```diff
@@ -22,6 +22,8 @@
 
   private long activityCount;
 
+  private long entryTemplateCount;
```

Comment:
If we are not going to display this on the main page, let's not add the extra query.